src/libs/actions/ReimbursementAccount/store.ts

```diff
@@ -46,7 +48,11 @@ function getBankAccountList() {
 }
 
 function hasCreditBankAccount() {
-    return _.some(bankAccountList, (bankAccountJSON) => {
+    if (!bankAccountList) {
+        return false;
+    }
+
+    Object.entries(bankAccountList).some(([, bankAccountJSON]) => {
```

Comment:
Why not Object.values here?

Reply:
Ah yeah, honestly I have no idea 😅 But in fact, values seem a better choice here.